src/modules/esl-utils/misc/format.ts

```diff
@@ -1,4 +1,4 @@
-import {get} from './object';
+import { get } from './object';
```

Comment:
Spaces should be warned by eslint
Something went wrong with our current eslint checker ...